go/teams/service_helper.go

```diff
@@ -478,3 +478,41 @@ func GetRootID(ctx context.Context, g *libkb.GlobalContext, id keybase1.TeamID)
 
 	return team.Name.RootAncestorName().ToTeamID(), nil
 }
+
+func ReAddMemberAfterReset(ctx context.Context, g *libkb.GlobalContext, teamname, username string) error {
```

Comment:
We don't want to get stuck passing team around `__keybase_implicit_team_` names. I think this function should take a teamID, and a UV.